### ISSUE TITLE
Color prompt

### ISSUE THREAD

**Issue by kensanata** [solderpunk/VF-1#3]
I'd like to see a color prompt so that it's easy to pick out what part of the screen is actually new.
[ANSI escape sequences](https://stackoverflow.com/questions/287871/print-in-terminal-with-colors) to the rescue?

**Comment by solderpunk** [comment]
Yeah, this would actually be nice, as sometimes it's hard to see where the line between consecutive short gopher items is.

I would prefer it if there was some user control over this, as some people may not want a coloured prompt.  This raises the issue of having some kind of config file...

I had been thinking of getting VF-1 to look for, say, `~/vf1.rc` and, if it exists, treating this as a script of VF-1 commands.  This would be a way to set a homepage (you could have a one line rc file of just `go phlogosphere.org` or whatever), or always start with your bookmarks open (if you were too lazy to add `--bookmarks` everytime).  This is convenient in that it is very light on code, as we don't have to parse a new file format of any kind, just read lines and pass them to the `Cmd` object.  It just means we then have to have any user configurable stuff doable from the regular prompt.  Maybe that's not so bad, we could just add a `do_set` command or something?

**Comment by solderpunk** [comment]
Some possibly very ugly colours are now in place! :)

**Comment by solderpunk** [comment]
Slightly less ugly now, and supposed to actually resemble a "real" VF-1.
